docs/changehistory/NextVersion.md

```diff
@@ -6,12 +6,17 @@ publish: false
 Table of contents:
 
 - [Electron 27 support](#electron-27-support)
+- Inserting,updating & deleting aspect require exclusive lock on the element that owns the aspect
 - [Display](#display)
 
 ## Electron 27 support
 
 In addition to [already supported Electron versions](../learning/SupportedPlatforms.md#electron), iTwin.js now supports [Electron 27](https://www.electronjs.org/blog/electron-27-0).
 
+## Inserting, updating or deleting an aspect now requires the exclusive lock be held on its element.
+
+This is a new requirement to prevent conflicts that may result in corrupt changesets. Before changes may be made to an element's aspects, you must now acquire its exclusive lock (see `IModelDb.LockControl.acquireLocks`).
```

Comment:
```suggestion
## Element aspects require locking

Inserting, updating or deleting an aspect now requires the exclusive lock be held on its element. This is a new requirement to prevent conflicts that may result in corrupt changesets. Before changes may be made to an element's aspects, you must now acquire its exclusive lock (see `IModelDb.LockControl.acquireLocks`).
```